modules/source/reactor-ip/config/reactor-ip.properties

```diff
@@ -1,2 +1,2 @@
 info.shortDescription = Reactor based source that allows a remote client to submit data over a raw TCP or UDP socket.
-options_class = org.springframework.xd.integration.reactor.net.NetServerSourceOptionsMetadata
+options_class = org.springframework.xd.integration.reactor.net.ReactorPeerSourceOptionsMetadata
```

Comment:
guess this should be ReactorNetSourceOptionsMetadata as I get the error 

```
Exception in thread "main" java.lang.IllegalStateException: Unable to load class used by ModuleOptionsMetadata: org.springframework.xd.integration.reactor.net.ReactorPeerSourceOptionsMetadata
```
when compiling (or it wasn't added)